stratum/hal/lib/tdi/tdi_table_manager.cc

```diff
@@ -473,17 +473,18 @@ ::util::StatusOr<::p4::v1::TableEntry> TdiTableManager::BuildP4TableEntry(
       param->set_param_id(expected_param.id());
       param->set_value(value);
     }
-  } else {
-    // Action profile member id
-    uint64 action_member_id;
-    if (table_data->GetActionMemberId(&action_member_id).ok()) {
-      result.mutable_action()->set_action_profile_member_id(action_member_id);
+  }
```

Comment:
Please restore the original indentation.

Google C++ coding style (used throughout all of Stratum) uses 2-space indentation.